src/main/webapp/i18n/de/global.json

```
@@ -35,6 +35,7 @@
                 "quizExercise": "Quiz Aufgabe",
                 "lecture": "Vorlesung",
                 "competency": "Kompetenz",
+                "ltiOutcomeUrl": "Lti Outcome Url",
```

```suggestion
                "ltiOutcomeUrl": "LTI-Outcome-URL",
```
probably like this, since I guess it is a fixed name as part of the definition.

LTI and URL should definitely be capitalised, since they are intialisms.